packages/pharos/src/components/tabs/pharos-tabs.test.ts

```diff
@@ -244,4 +244,27 @@ describe('pharos-tabs', () => {
     await component.updateComplete;
     expect(count).to.equal(0);
   });
+
+  it('does not render panel separator', async () => {
+    const spanElement = component.renderRoot.querySelector('.panel-separator') as HTMLDivElement;
+
+    expect(spanElement).to.be.null;
+  });
+
+  it('renders panel separator', async () => {
+    component = await fixture(html`
+      <pharos-tabs panelseparator="true">
```

Comment:
Does this also work with `panel-separator` as two words? That would be my expectation so want to make sure I'm not missing anything.

Reply:
Ah, I think @Niznikr's suggestion below does the trick.